sentry-android-fragment/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+
+plugins {
+    id("com.android.library")
+    kotlin("android")
+    jacoco
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+android {
+    compileSdkVersion(Config.Android.compileSdkVersion)
+
+    defaultConfig {
+        targetSdkVersion(Config.Android.targetSdkVersion)
+        minSdkVersion(Config.Android.minSdkVersionFragment)
+
+        versionName = project.version.toString()
+        versionCode = project.properties[Config.Sentry.buildVersionCodeProp].toString().toInt()
+
+        // for AGP 4.1
+        buildConfigField("String", "VERSION_NAME", "\"$versionName\"")
+    }
+
+    buildTypes {
+        getByName("debug")
+        getByName("release")
+    }
+
```

Comment:
missing the `testOptions` block here, I recall having troubles if not setting them

Reply:
Gotcha, I've removed it initially as those seemed not needed at a first glance, thanks for noticing. Added in 380b732a4181b3ca4efa041a665ca40fbc999e04

Reply:
it was more of a problem when running `@RunWith(AndroidJUnit4::class)`, so lets keep it for the sake of safety